### DISCUSSION TITLE
whole-body-controllers replaced WBI-Toolbox-Controllers

### DISCUSSION THREAD

**Discussion by gabrielenava** [robotology/community#288] [Announcements]
Following the recent release of [WBToolbox 3.0](https://github.com/robotology/QA/issues/279), also the Simulink torque controllers for balancing and walking (that use WBToolbox) have been updated. The new controllers are stored in the new repository [whole-body-controllers](https://github.com/robotology/whole-body-controllers).  To download and install it, you can follow the [instructions](https://github.com/robotology/whole-body-controllers/blob/master/README.md) provided in the readme file. It is also possible to install this repository through the [robotology-superbuild](https://github.com/robotology/robotology-superbuild) (enable the `ROBOTOLOGY_ENABLE_DYNAMICS` option).

**The previous repository containing the controllers [WBI-Toolbox-Controllers](https://github.com/robotology-playground/WBI-Toolbox-controllers) is no more maintained**. If you previously installed `WBIToolboxControllers` through the `robotology-supebuild`, it is suggested to manually remove it from the `robotology-superbuild/robotology` folder.

**Comment by traversaro** [comment]
cc @robotology/codyco-developers
